ROADMAP.md

```diff
@@ -17,16 +17,18 @@
 
 ## Enhancements
 
-### Hyperparameter Tuning
+### Common
 
 - Delete Suggestion deployment after Experiment is finished [#1061](https://github.com/kubeflow/katib/issues/1061)
 - Save Suggestion state after deployment is deleted [#1062](https://github.com/kubeflow/katib/issues/1062)
 - Reconsider the design of Trial Template [#906](https://github.com/kubeflow/katib/issues/906)
+- Support new Kubeflow operators (MXNet, XGBoost, MPI)
```

Comment:
Can we come up with an extensible model for supporting different operators? Maybe rephrase this

* Design an extensible model for integrating with custom resources

Reply:
+1

Reply:
Agree, updated.